### ISSUE TITLE
Incorrectly Rotated Composites in Cyrillic

### ISSUE THREAD

**Issue by dberlow** [googlefonts/googlesans#242]
Both Roman and Italic include 2 composite glyphs that are rotated. This has caused some issues before with Google testing.
	    (Roman : GID 669 & 694, Italic: GID 524 & 549)

**Comment by dberlow** [comment]
In all 8 source masters, 

Glyph ID 669 = uni04BA Cyrillic Capital Letter Shha
needs to be decomposed 
then 
Glyph ID 694 = uni0526 Cyrillic Capital Letter Shha with Descender
a composite, needs to be composed with the glyph above, and "descender-cy", as it is now. 

Google Sans Roman

Glyph ID 669 = uni04BA Cyrillic Capital Letter Shha
Glyph ID 694 = uni0526 Cyrillic Capital Letter Shha with Descender

Google Sans Italic

Glyph ID 524 = uni04BA Cyrillic Capital Letter Shha
Glyph ID 549 = uni0526 Cyrillic Capital Letter Shha with Descender

**Comment by chrissimpkins** [comment]
Thanks David!  I am going to move this issue to the googlefonts/googlesans repository so that we can track it there.  Will have a look into fixing this as soon as we are merged.

**Comment by chrissimpkins** [comment]
@dberlow Are you able to change the outlines / composites downstream?  If so, I'll pull in the changes and add them to the branch that Nikolaus is working on once we have all of the anchors in place.  If anchors are involved here, this will require some manual cut/paste on our end.  Please point to the commit that you push with the changes.  Thanks!

**Comment by moyogo** [comment]
There's a fontmake/ufo2ft filter to decompose transformed (scaled, rotated or flipped) components at build time: DecomposeTransformedComponentsFilter.

You can add it to each UFO with something like that:
```bash
python3 -c 'import sys; from ufoLib2 import Font; path = sys.argv[-1]; font = Font(path); filtersKey = "com.github.googlei18n.ufo2ft.filters"; filters = font.lib.get(filtersKey, []); filters.append({"name": "DecomposeTransformedComponentsFilter", "pre": True}); font.lib[filtersKey] = filters; font.save()' <UFO path>
```

With the next version of ufo2ft there will be a command line tool to apply a filter to a UFO if you'd rather apply this to the sources than at build time:
```
 python3 -m ufo2ft.filters --output <UFO path>  <UFO path> DecomposeTransformedComponentsFilter
```
or currently with the following:
```
python3 -c 'import sys; from ufoLib2 import Font; from ufo2ft.filters import DecomposeTransformedComponentsFilter; path = sys.argv[-1]; font = Font(path); philter = DecomposeTransformedComponentsFilter(); print("\n".join(sorted(philter(font)))); font.save()'
```

**Comment by chrissimpkins** [comment]
Thank you very much Denis!  As I understand, this decomposes all transformed components in the sources correct?  

Is there a situation where it is desirable to keep any of them?  What is the issue with transformed components?

**Comment by chrissimpkins** [comment]
> Thank you very much Denis!  As I understand, this decomposes all transformed components in the sources correct?  
> 
> 
> 
> Is there a situation where it is desirable to keep any of them?  What is the issue with transformed components?

Question answered.  Ignore.

**Comment by chrissimpkins** [comment]
D Berlow changes merged in b26739deb19ca6740903a69df7499c8851ff868a

**Comment by chrissimpkins** [comment]
Visually confirmed changes in Font Goggles.  LGTM.  #240 will fix this issue.
